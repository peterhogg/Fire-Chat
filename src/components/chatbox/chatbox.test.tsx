import React from 'react';
import ReactDOM from 'react-dom';
import Chatbox from './index';
import renderer from 'react-test-renderer';

describe('chatbox', () =>{
	
	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<Chatbox />, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	it('has a valid snapshot', () =>{
		const component = renderer.create(
			<Chatbox />
		);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	})

})
