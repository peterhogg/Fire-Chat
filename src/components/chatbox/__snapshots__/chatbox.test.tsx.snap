// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chatbox has a valid snapshot 1`] = `
<div
  className="chatbox"
>
  <input
    type="textbox"
  />
  <button
    type="submit"
  >
    Send
  </button>
</div>
`;
